projects.md

```diff
@@ -2,33 +2,44 @@
+  * [Wireless Warrington](/blog/2014/10/05/wireless_warrington_begins) is a part of
 PittMesh that was funded by a grant from the [Urban Redevelopment Authority of
 Pittsburgh](http://www.ura.org/) facilitated by the [Mount Washington Community
-Development Corporation](http://mwcdc.org/). This network has joined the main
-network.
+Development Corporation](http://mwcdc.org/). This PittMesh segment is considered the "main" segment and is also closest to our offices at the Hardware Store, a coworking space in Allentown.
+
+## The Meta Mesh Webstore
+
+There are two varieties of mesh networking devices available today. You can purchase an all-in-one, networking-as-a-service devices that cost hundreds of dollars each and often require and Internet connection and a yearly license. You can also take the DIY route, buy a supported OpenWRT router, flash the firmware, download the packages, and configure each device by hand. Each solution is daunting. You either need significant financing or significant knowledge to accomplish the respective route.
+
+Meta Mesh is combining the simplicity of purchasing a powerful, enterprise-grade device with the flexibility of a OpenWRT-based firmware, preconfigured for ease-of-use. Launching in September, 2015, the Community Wireless Networking Webstore will be the first of its kind, complete with kit-based purchases and a la carte options. Each device will be pre-assembled and pre-configured by Meta Mesh. All you need to do is mount it and plug it in.
+
+A launch date for the webstore will be announced shortly.
 
-[PittMesh: Morningside](#) is a network currently in planning stages. Parties
-interested in hosting a node [should fill out this
-form](http://goo.gl/forms/wbRUJdfpAx).
+## Meta Mesh Grant Writing
 
+  While PittMesh can be built one node at a time, seed networks spur faster adoption. Seed networks are funded by grants or generous residents who want to accelerate growth of PittMesh in specific areas of the city. Meta Mesh will work with your community to write a grant on behalf of a 501c3 entity. Since Meta Mesh is an limited liability company, we are not eligible to receive grants directly. Meta Mesh will direct you on how to properly equip a local team from your community with the installation tools, skills, and wireless equipment to build your own network and connect it to the greater PittMesh network. While some of these seed networks may be isolated from other parts of the PittMesh network, once online, we actively seek to connect these network segmemts to the main network either physically or temporarily over the Internet. Meta Mesh holds training classes for your community's installation team to prepare them to install, maintain, and expand the network independent of Meta Mesh. If you don't rely on Meta Mesh to manage your network, you further decentralize who owns and controls the network just like decentralizing the infrastructure itself!
```

Comment:
> 501c3

501(c)3, to be pedantic.

Probably should say "recognized 501(c)3 charitable organization".